enhancements/user-monitoring.md

```diff
@@ -0,0 +1,248 @@
+---
+title: user-monitoring
+authors:
+  - "@s-urbaniak"
+  - "@brancz"
+reviewers:
+  - "@sichvoge"
+  - "@openshift/openshift-team-monitoring"
+approvers:
+  - "@sichvoge"
+  - "@openshift/openshift-team-monitoring"
+creation-date: 2019-09-19
+last-updated: 2019-09-19
+status: implementable
+---
+
+# user-monitoring
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
```

Comment:
We can mark this as well?

Reply:
hmm, i think we did not specify the actual _criteria_ for GA.